test/arrays/ferguson/semantic-examples/valgrind-futures/4-return-alias-local.chpl

```diff
@@ -1,7 +1,7 @@
 use samples;
 proc f() {
   var A:[1..4] R;
-  var B => A;
+  ref B = A;
```

Comment:
That test is a different case than this one, isn't it?  That one returns a local array using `ref` return intent, so its .good file contains an error message.  Whereas this one returns a `ref` to a local array using default intent (which results in a copy out, hopefully with no leaks or valgrind issues when it's working well).